src/Tribe/Customizer/Global_Elements.php

```diff
@@ -91,16 +150,8 @@ public function get_css_template( $template ) {
 				.tribe-events .datepicker .month.current.focused,
 				.tribe-events .datepicker .year.current:hover,
 				.tribe-events .datepicker .year.current:focus,
-				.tribe-events .datepicker .year.current.focused,
-				.tribe-common .tribe-common-c-loader__dot {
-					background: rgba( ' . $accent_css_rgb . ', 0.07);
-				}
-			';
-
-			$template .= '
-				.tribe-events .tribe-events-calendar-month__day--current .tribe-events-calendar-month__day-date-link:hover,
-				.tribe-events .tribe-events-calendar-month__day--current .tribe-events-calendar-month__day-date-link:focus {
-					color: rgba( ' . $accent_css_rgb . ', 0.8);
+				.tribe-events .datepicker .year.current.focused {
+					background: ' . $accent_color_background .';
```

Comment:
:no_entry_sign: Concat operator must be surrounded by a single space

<sub>hash: d0dd56bfbf1c23d945fd7a4155e39e37</sub>